### PR TITLE
[Email] add advanced ut

### PR DESCRIPTION
**Pull request by DingmaomaoBJTU**
## Description
1.add advanced ut. covers previous/next page, show 0/1/1+ emails, show 0/1/1+ contacts and many more.
Coverage for now is 60%. 
2. Fix paging merge issue
3. Update some resource string.

## Related Issue
[Email] Support more UT cases #403

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the appropriate unit tests
- [ ] I have updated related documentation

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
- [ ] A duplicate issue is filed to track future  work.

If you have updated responses or `.lu` files:
- [ ] All languages have been updated
- [ ] You have tested deployment with your new models

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/Shared/Resources/EmailSharedResponses.json [hunk 1]
```diff
@@ -237,12 +237,12 @@
   "EmailNotFound": {
     "replies": [
       {
-        "text": "You have no new email.",
-        "speak": "You have no new email."
+        "text": "You have no relative email.",
```
Comment (lauren-mills) [inline solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/Shared/Resources/EmailSharedResponses.json]: Should this be "relevant"?

File: solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json [hunk 2]
```diff
@@ -1,54 +1,50 @@
...
-      }
-    ],
-    "inputHint": "acceptingInput"
-  },
   "ReadOutPrompt": {
     "replies": [
       {
-        "text": "Which one do you want to hear?",
-        "speak": "Which one do you want to hear?"
+        "text": "Which one do you want to hear? You can also forward, reply or delete any of this.",
```
Comment (lauren-mills) [inline solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json]: "You can also forward, reply or delete any of **these**."

**Review by lauren-mills — CHANGES_REQUESTED** [review]

File: solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json [hunk 3]
```diff
@@ -1,54 +1,50 @@
 {
-  "EmailNotFound": {
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json:2]: why remove EmailNotFound?

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json (hunk 3 above)
Comment (DingmaomaoBJTU) [reply solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json:2]: This is a dup resource. In code, we use the "EmailNotFound" from EmailSharedResponses.json, So remove this.

File: solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json (hunk 2 above)
Comment (DingmaomaoBJTU) [reply solutions/Virtual-Assistant/src/csharp/skills/emailskill/Dialogs/ShowEmail/Resources/ShowEmailResponses.json]: Thanks!!!

**Review by lauren-mills — APPROVED** [review]

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**